src/test/java/com/cloudbees/hive/http/BeeAPITest.java

```diff
@@ -61,6 +63,7 @@ public void shouldBeAbleToCreateANewBee() throws Exception {
             .contentType(MediaType.APPLICATION_JSON)
             .content(objectMapper.writeValueAsBytes(maya)))
             .andExpect(status().isCreated())
-            .andExpect(header().string("Location", Matchers.endsWith(maya.getId())));
+            .andExpect(header().string("lat", Matchers.endsWith(maya.getLat().toString())))
```

Comment:
I don't get this. The location is the URI to access the `Bee` we just created. I'm removing this.